src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -57,6 +58,10 @@ protected static ConfigDef baseConfigDef() {
           .define(MAX_IN_FLIGHT_REQUESTS_CONFIG, Type.INT, 5, Importance.MEDIUM,
                   "The maximum number of indexing requests that can be in-flight to Elasticsearch before blocking further requests.",
                   group, 5, Width.SHORT, "Max In-flight Requests")
+          .define(NODE_DISCOVERY_FREQUENCY, Type.LONG, 60, Importance.MEDIUM,
```

Comment:
Hmm, is there going to be an equivalent in every library? This seems like it exposes implementation details. For example, if we decided to switch to the [Java REST client maintained alongside ES](https://www.elastic.co/guide/en/elasticsearch/client/java-rest/current/index.html), which didn't exist when we initially implemented this, would there be something equivalent we could use?

More generally, I'm a bit curious about the use of such a config. If you need this to handle your cluster's nodes moving to different hosts, doesn't this mean your application would break if you, e.g., had a process failure and had to restart from the original config? If not, doesn't that imply you already have some sort of discovery mechanism (DNS, load balancer, etc) that handles this for you? Or am I misunderstanding the use case for node discovery?